app/controllers/users_controller.rb

```diff
@@ -47,4 +48,8 @@ def user_parameters
                                  :spire,
                                  :staff
   end
+
+  def allow_only_admin
+    deny_access && return unless @current_user.admin?
```

Comment:
Interestingly, redirect_to as written in @dfaulken's link *has* to be `and`, while switching to `&&` is okay-ish here (but I like `and` better) and I think RuboCop knows the difference.

```ruby
redirect_to unauthenticated_session_path and return
# is the same as
redirect_to(unauthenticated_session_path) and return
# while
redirect_to unauthenticated_session_path && return
# is the same as
redirect_to(unauthenticated_session_path && return)
```